### ISSUE TITLE
Feature Request: Select Object Printing Order for Dual Extrusion Prints

### ISSUE THREAD

**Issue by bethebeer** [alexrj/Slic3r#2415]
I have been printing two material prints with two extruders.  One material is PVA and the other is a flexible filament. The PVA is then dissolved away after the print.  When The PVA prints first, the flexible plastic prints nicely; non contiguous parts have nice perimeters and no strings connecting them to each other.  When the flexible filament prints first, strings form between non contiguous parts. No matter how much retraction, or at what speed or temperature, the flexible filament will not break cleanly when going from one part to another unless the PVA on that layer is already printed.

It would be great if there was a way to override the more efficient path for the option that prints support first. Or in the case of manual support, prints a specific part, or uses a specific extruder first, each layer.

**Comment by alexrj** [comment]
So are you placing PVA loops all around the perimeters of your flexible filament? Are you you basically creating a mould for extruding the flexible filament?

**Comment by bethebeer** [comment]
Yes, it is basically creating a mold for the flexible filament.  I am subtracting the model (that will be printed in flexible filament) from the smallest confining cube and printing the cube in the dissolvable support.
